packages/block-library/src/paragraph/edit.js

```diff
@@ -48,6 +56,27 @@ function ParagraphRTLToolbar( { direction, setDirection } ) {
 	) );
 }
 
+function useDropCapMinimumHeight( isDropCap ) {
+	const [ minimumHeight, setMinimumHeight ] = useState();
+	useEffect(
+		() => {
+			const element = querySelector && querySelector( PARAGRAPH_DROP_CAP_SELECTOR );
```

Comment:
`querySelector` shouldn't ever be `undefined` since you are binding it in the top scope.